server/index.yaml

```diff
@@ -0,0 +1,20 @@
+indexes:
+
+- kind: Commit
+  properties:
+  - name: customerId
+  - name: commitStatus
+  - name: createdAt
+    direction: desc
+
+- kind: Commit
+  properties:
+  - name: listingId
+  - name: commitStatus
+  - name: createdAt
+
+- kind: Listing
+  properties:
+  - name: merchantId
+  - name: listingStatus
+  - name: deadline
```

Comment:
do you need 'direction: desc' for deadline?

Reply:
@ifhuang Hmm I think I prefer the default direction, which is 'asc' so that the one with the nearest deadline will appear first for ongoing listings. And for successful listings that require the merchant to deliver the items, I also want the oldest one to appear first, so that the merchant will send these items first.
I guess for completed and unsuccessful listings, it might be good to have 'direction: desc' but I think that's not as important and can be added later on when needed.

Reply:
@ifhuang I prefer the default direction which is ascending. For 'ongoing' listings, I would like to display the ones with the nearest deadline first, as that seems to be the most urgent. For 'successful' listings, I would like to display the oldest ones first, so that the merchant will send the items for these listings first (so that the customers don't have to wait for too long).